src/components/organisms/SfBottomNavigation/SfBottomNavigation.md

```diff
@@ -0,0 +1,52 @@
+# SfBottomNavigation
+
+Mobile-only bottom navigation menu with icon elements and floating Button.
+
+## HTML Template
```

Comment:
Remove html, now we use different template (hTML is now automatically added to info panel) + storybook can't parse it in this form ;/